watch.mjs

```diff
@@ -0,0 +1,42 @@
+import fsCallback from 'node:fs';
+import { buildComponentSets, listComponentSetPaths, validateComponentSet } from './build.mjs';
+
+export async function watchComponentSets() {
+    // Watch does not rebuild vendor.js and design.css
+    await buildComponentSets();
```

Comment:
This also creates the zip in the `dist` folder, I guess that's not what we want?